planet/cli/orders.py

```diff
@@ -253,11 +253,26 @@ async def create(ctx, request: str, pretty):
               default=False,
               is_flag=True,
               help='Send email notification when order is complete.')
+@click.option('--archive-type',
+              type=click.Choice(['zip']),
+              help="Optionally zip archive each item bundle.")
+@click.option('--archive-filename',
+              default='{{name}}-{{order_id}}.zip',
+              show_default=True,
+              help="Templated filename for archived bundles or orders.")
+@click.option('--single-archive',
+              is_flag=True,
+              default=False,
+              show_default=True,
+              help="Optionally zip archive all item bundles together.")
 @click.option(
     '--cloudconfig',
+    '--delivery',
     type=types.JSON(),
-    help="""Credentials for cloud storage provider to enable cloud delivery of
-    data. Can be a json string, filename, or '-' for stdin.""")
+    help=("Credentials for cloud storage provider to enable cloud delivery of "
+          "data. Can be a json string, filename, or '-' for stdin. "
+          "Optionally, archive parameters can be set using the same JSON. "
+          "The --delivery option is an alias for this use case."))
```

Comment:
Too confusing? Or okay?

Reply:
If feels a little unclear as is. I would reword such that --cloudconfig is the alias, and merge the archive parameter line into the first sentence. E.g.:
```
"Delivery configuration, which may include credentials for a cloud storage "
"provider, to enable cloud delivery of data, and/or parameters for bundling "
"deliveries as zip archives. Can be a json string, filename, or '-' for stdin. "
"The --cloudconfig option is an alias for this use case."
```